From b89c14599fd8798eef64431bb05e8705aaaea683 Mon Sep 17 00:00:00 2001
From: Emanuel Haupt <ehaupt@critical.ch>
Date: Mon, 6 Mar 2023 10:42:33 +0100
Subject: [PATCH] Restore includedir and libdir

Restore includedir and libdir. This was removed some time between 0.6.2 and 0.6.3 [1] causing pkg-config --libs and --ldflags to report empty -I and -L paths.

[1] https://github.com/metabrainz/libdiscid/compare/v0.6.2...v0.6.3

See also:
https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=269990
---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1f24916..911bf4a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -38,6 +38,8 @@ SET(INCLUDE_INSTALL_DIR ${CMAKE_INSTALL_FULL_INCLUDEDIR} CACHE PATH "Installatio
 SET(PACKAGE ${PROJECT_NAME})
 SET(VERSION ${PROJECT_VERSION})
 SET(PACKAGE_STRING "${PACKAGE} ${VERSION}")
+SET(includedir ${INCLUDE_INSTALL_DIR})
+SET(libdir ${LIB_INSTALL_DIR})
 
 CONFIGURE_FILE(libdiscid.pc.in libdiscid.pc)
 CONFIGURE_FILE(versioninfo.rc.in versioninfo.rc)
